simd/src/Kokkos_SIMD_Scalar.hpp

```diff
@@ -412,6 +523,17 @@ template <class T>
   return a == basic_simd_mask<T, Kokkos::Experimental::simd_abi::scalar>(false);
 }
 
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
+template <class T>
+[[nodiscard]] KOKKOS_DEPRECATED KOKKOS_FORCEINLINE_FUNCTION T
```

Comment:
Would you mind replacing `KOKKOS_DEPRECATED` with `KOKKOS_DEPRECATED_WITH_COMMENT` like you do in some overloads?

Reply:
Added deprecation messages for `hmin` and `hmax`.